src/Engines/Extensions/engine.cubeTexture.ts

```diff
@@ -220,16 +220,17 @@ ThinEngine.prototype._partialLoadImg = function (url: string, index: number, loa
     }
 };
 
-ThinEngine.prototype._setCubeMapTextureParams = function (texture: InternalTexture, loadMipmap: boolean, maxLevel: number): void {
+ThinEngine.prototype._setCubeMapTextureParams = function (texture: InternalTexture, loadMipmap: boolean, maxLevel?: number): void {
     var gl = this._gl;
     gl.texParameteri(gl.TEXTURE_CUBE_MAP, gl.TEXTURE_MAG_FILTER, gl.LINEAR);
     gl.texParameteri(gl.TEXTURE_CUBE_MAP, gl.TEXTURE_MIN_FILTER, loadMipmap ? gl.LINEAR_MIPMAP_LINEAR : gl.LINEAR);
     gl.texParameteri(gl.TEXTURE_CUBE_MAP, gl.TEXTURE_WRAP_S, gl.CLAMP_TO_EDGE);
     gl.texParameteri(gl.TEXTURE_CUBE_MAP, gl.TEXTURE_WRAP_T, gl.CLAMP_TO_EDGE);
     texture.samplingMode = loadMipmap ? Constants.TEXTURE_TRILINEAR_SAMPLINGMODE : Constants.TEXTURE_LINEAR_LINEAR;
 
-    if (loadMipmap && this.getCaps().textureMaxLevel && maxLevel > 0) {
+    if (loadMipmap && this.getCaps().textureMaxLevel && maxLevel !== undefined && maxLevel > 0) {

```

Comment:
I think maxLevel == 0 is a valid value (and in effect disable mipmapping).

Reply:
I should rename maxLodLevel... I took it from the previous PR to ensure the PG with incomplete DDS was ok in WEBGPU as well :-( maxLodLevel is containing the value + 1

Reply:
I agree that maxLevel should be set if set to 0 but I'm not so sure for the name change

Reply:
We need to test to be sure 0 is ok :) I found no doc on that topic